applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -1012,11 +1632,19 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	SHELL_COND_CMD(CONFIG_SHELL, packing, NULL, "Set type of packing", cmd_packing),
 	SHELL_COND_CMD(CONFIG_SHELL, preset, NULL, "Set preset", cmd_preset),
 	SHELL_COND_CMD(CONFIG_SHELL, lang, NULL, "Set language", cmd_lang_set),
+	SHELL_COND_CMD(CONFIG_SHELL, immediate, NULL, "Set immediate rendering flag",
+		       cmd_immediate_set),
 	SHELL_COND_CMD(CONFIG_SHELL, num_subgroups, NULL, "Set number of subgroups",
 		       cmd_num_subgroups),
 	SHELL_COND_CMD(CONFIG_SHELL, num_bises, NULL, "Set number of BISes", cmd_num_bises),
 	SHELL_COND_CMD(CONFIG_SHELL, context, NULL, "Set context", cmd_context),
 	SHELL_COND_CMD(CONFIG_SHELL, location, NULL, "Set location", cmd_location),
+	SHELL_COND_CMD(CONFIG_SHELL, name, NULL, "Set broadcast name", cmd_name),
```

Comment:
```suggestion
	SHELL_COND_CMD(CONFIG_SHELL, broadcast_name, NULL, "Set broadcast name", cmd_name),
```
I think it's better to have it more specific and on the same format as `adv_name`